docs/config.md

```diff
@@ -9,3 +9,11 @@ servers if there are any uncommitted or untracked files in your git repository.
 the `disable_commit_guard` configuration option will disable this protection.
 
     config.orchestrate.disable_commit_guard = true
+
+## `take_synced_folder_ownership`
+
+When multiple users are using Vagrant Orchestrate to push to the same target servers,
+there can arise permission issues for folders that are synced. When `true`, Vagrant
+Orchestrate will change ownership of the guestpath of all synced folders to be the
+`owner` specified in the `synced_folder` or the `ssh_info.username`. Has no affect
```

Comment:
Can we get a grammarian to weight on on affect vs effect here? I'm not confident enough to actually say that I think it's supposed to be 'effect' here.